documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -3156,9 +3156,15 @@ endif::community[]
 
 |[[sqlserver-property-skipped-operations]]<<sqlserver-property-skipped-operations, `+skipped.operations+`>>
 |`t`
-|A comma-separated list of operation types that will be skipped during streaming.
-The operations include: `c` for inserts/create, `u` for updates, `d` for deletes, `t` for truncates, and `none` to not skip any operations.
-By default, truncate operations are skipped (not emitted by this connector).
+|A comma-separated list of the operation types that you want the connector to skip during streaming.
+You can configure the connector to skip the following types of operations:
+
+* `c` (insert/create)
+* `u` (update)
+* `d` (delete)
+* `t` (truncate)
```

Comment:
This is the same case as Db2.

Reply:
@jpechane Similar to the Db2 case, as I look at the SQL Server 2022 documentation, I find a reference to using the `TRUNCATE TABLE` command. Should the connector doc state that the Debezium SQL Server connector does not support `truncate` operations vs. SQL Server does not support `truncate` events?